index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="index.css" />
```

Comment:
- [ ] [OPTIONAL] Good job 🔥👏. I recommend moving the link in the credit section to the Acknowledgments part in the readme file file 📘

Screenshot
![image](https://user-images.githubusercontent.com/69011963/160567794-61928e6c-e684-4b7b-8a64-1ccf6f735750.png)